integration/docker_engine_test.go

```diff
@@ -236,7 +236,7 @@ func TestRun_WithDockerEngine(t *testing.T) {
 			name:          "sad path, invalid image",
 			invalidImage:  true,
 			testfile:      "badimage:latest",
-			expectedError: "unable to inspect the image (index.docker.io/library/badimage:latest): Error: No such image: index.docker.io/library/badimage:latest",
+			expectedError: "unable to inspect the image",
```

Comment:
I think it's good to be specific. The rest of the error includes the name of the image as well, which in this case is important.

Reply:
Roger that 👍  I've added the image name.